rpc/lib/client/ws_client_test.go

```diff
@@ -194,7 +194,7 @@ func TestNotBlockingOnStop(t *testing.T) {
 	}
 }
 
-func startClient(t *testing.T, addr net.Addr) *WSClient {
+func startClient(t *testing.T, addr fmt.Stringer) *WSClient {
```

Comment:
I think we want net.Addr here, not `fmt.Stringer`

Although, maybe we should change it to 

```suggestion
func startClient(t *testing.T, addr string) *WSClient {
```